src/charts/donut.js

```diff
@@ -241,23 +242,38 @@ define(function(require){
                     .selectAll('g.arc')
                     .data(layout(data));
 
-                slices.enter()
-                  .append('g')
-                    .each(storeAngle)
-                    .each(reduceOuterRadius)
-                    .classed('arc', true)
-                    .on('mouseover', handleMouseOver)
-                    .on('mouseout', handleMouseOut)
-                .merge(slices)
-                  .append('path')
-                    .attr('fill', getSliceFill)
-                    .on('mouseover', tweenGrowthFactory(externalRadius, 0))
-                    .on('mouseout', tweenGrowthFactory(externalRadius - radiusHoverOffset, pieHoverTransitionDuration))
-                    .transition()
-                    .ease(ease)
-                    .duration(pieDrawingTransitionDuration)
-                    .attrTween('d', tweenLoading);
-
+                if (isAnimated) {
+                    slices.enter()
```

Comment:
You mean the first block, before the merge?

Let me try, but I think it won't work...